Add instructions for tracking superset changes via git (#1511)
medic/cht-docs: content/en/building/guides/data/analytics/setup.md

To remove all the data volumes, add `-v` at the end of this command.

## Setup Superset

> To build data visualization dashboards, follow the [Superset instructions](https://superset.apache.org/docs/installation/installing-superset-using-docker-compose/) to run Superset and connect it to the PostgreSQL database. Medic recommends adding a way to track Superset changes via a git repository or any other version control system to make it easier to track changes over time and potentially catch and remediate bugs and regressions. Instruction on how to do this using a Github action can be found [here](https://github.com/medic/cht-sync/blob/main/.github/actions/superset-backup/README.md).

Reviewer: ```suggestion
To build data visualization dashboards, follow the [Superset instructions](https://superset.apache.org/docs/installation/installing-superset-using-docker-compose/) to run Superset and connect it to the PostgreSQL database. It is recommended that a way to track Superset changes be added via a git repository or any other version control system to make it easier to track changes over time and potentially catch and remediate bugs and regressions. Instruction on how to do this using a Github action can be found [here](https://github.com/medic/cht-sync/blob/main/.github/actions/superset-backup/README.md).
```

Reviewer: Small suggestion to remove `Medic` mention.
